src/containers/b2b/RequisitionList.tsx

```diff
@@ -103,15 +88,62 @@ class RequisitionList extends Component<CartCreateProps, CartCreateState> {
     this.handleChange = this.handleChange.bind(this);
     this.clearListNameField = this.clearListNameField.bind(this);
     this.handleSaveList = this.handleSaveList.bind(this);
+    this.addToSelectedCart = this.addToSelectedCart.bind(this);
   }
 
   componentDidMount() {
+    this.loadRequisitionListData();
+  }
+
+  loadRequisitionListData() {
+    this.setState({ isLoading: true });
+    login()
+      .then(() => {
+        cortexFetch(`?zoom=${listsZoomArray.sort().join()}`, {
+          headers: {
+            'Content-Type': 'application/json',
+            Authorization: localStorage.getItem(`${Config.cortexApi.scope}_oAuthToken`),
+          },
+        })
+          .then(res => res.json())
+          .then((res) => {
+            if (res._itemlistinfo[0] && res._itemlistinfo[0]._allitemlists[0]) {
```

Comment:
Need to first check if res._itemlistinfo is present
```suggestion
            if (res._itemlistinfo && res._itemlistinfo[0] && res._itemlistinfo[0]._allitemlists[0]) {
```